packages/react-component-library/src/components/SelectE/SelectE.test.tsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { fireEvent, render, RenderResult } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { selectors } from '@defencedigital/design-tokens'
+
+import { SelectE } from '.'
+import { SelectEOption } from './SelectEOption'
+
+const { color } = selectors
+
+describe('SelectE', () => {
+  let onChangeSpy: (value: string) => void
+  let wrapper: RenderResult
+
+  describe('default props', () => {
```

Comment:
Would be nice to use the 'when...' style for the `describe()` here. e.g.

```suggestion
  describe('when using the default prop values', () => {
```